analyzers/src/SonarAnalyzer.CSharp/Rules/UnusedPrivateMember.cs

```diff
@@ -32,7 +31,7 @@ public sealed class UnusedPrivateMember : SonarDiagnosticAnalyzer
         private const string S4487DiagnosticId = "S4487";
         private const string S4487MessageFormat = "Remove this unread {0} field '{1}' or refactor the code to use its value.";
 
-        private static readonly DiagnosticDescriptor RuleS1144 = DescriptorFactory.Create(S1144DiagnosticId, S1144MessageFormat, fadeOutCode: true);
+        private static readonly DiagnosticDescriptor RuleS1144 = DescriptorFactory.Create(S1144DiagnosticId, S1144MessageFormat);
         private static readonly DiagnosticDescriptor RuleS4487 = DescriptorFactory.Create(S4487DiagnosticId, S4487MessageFormat, fadeOutCode: true);
```

Comment:
I see an inconsistency here between S1144 and S4487 which will be hard to explain later on.

I don't have a strong opinion on any decision, but I would favor consistency. I checked and we don't use the fadeOutCode much in the code, so we wouldn't have a guideline.

- you can see what makes more sense inside the IDE for both rules an take a consistent decision
- Or write a comment with an argument on why S1144 is different from S4487

Reply:
I checked out both in SonarLint. Then I disabled the fade-out effect for both rules.
We can turn them back in a more accurate way with https://github.com/SonarSource/sonar-dotnet/issues/6697